frontend/src/routes/Governance/common/util.tsx

```diff
@@ -650,18 +655,108 @@ export function hasInformOnlyPolicies(items: Array<PolicyTableItem>): boolean {
   return true
 }
 
+// Transform string annotation to key and value pair type
+/* istanbul ignore next */
```

Comment:
Why does this need to be ignored by istanbul?

Reply:
For unit tests,  All default functions from window (document) turn to wired words.